### ISSUE TITLE
"Exception: Error decoding frame: attribute unknown or not implemented..." when calling decompile_replay on recent replay (2022-02-12 06:20 UTC)

### ISSUE THREAD

**Issue by beeftendon** [SaltieRL/carball#271]
The replay in question can be found [here](https://ballchasing.com/replay/75cf0ba0-c0f6-4ae7-a696-f6c4d62b69a9).

```
import carball
carball.decompile_replay('/path/to/75cf0ba0-c0f6-4ae7-a696-f6c4d62b69a9.replay')
```

Raises the following exception:

```
Exception                                 Traceback (most recent call last)
<ipython-input-2-d4e98c6244c3> in <module>
      4 from carball.analysis.analysis_manager import AnalysisManager
      5 
----> 6 replay_json = cb.decompile_replay('/path/to/75cf0ba0-c0f6-4ae7-a696-f6c4d62b69a9.replay')
      7 
      8 # _json is a JSON game object (from decompile_replay)

~/.virtualenvs/rocketleague/lib/python3.6/site-packages/carball/decompile_replays.py in decompile_replay(replay_path)
     21     with open(replay_path, 'rb') as f:
     22         buf = f.read()
---> 23     return parse_replay(buf)
     24 
     25 

Exception: Error decoding frame: attribute unknown or not implemented: actor id / actor object id / attribute id: 36 / 288 / 24. found attribute TAGame.CarComponent_Dodge_TA:DodgeImpulse (unknown to boxcars) on Archetypes.CarComponents.CarComponent_Dodge in network cache data. This is likely due to a rocket league update or an atypical replay. File a bug report! Context: on frame: 133, last updated actor: (actor stream id / object id / name: 36 / 288 / Archetypes.CarComponents.CarComponent_Dodge, attribute stream id / object id / name: 22 / 276 / TAGame.CarComponent_TA:ReplicatedActive, attribute: Byte(3))
```

On carball 0.7.5, boxcars-py 0.1.13

**Comment by beeftendon** [comment]
Actually just noticed that the latest version of boxcars-py is actually 0.1.14. However, not sure why my `pip install` only sees up to 0.1.13. (I also realize this may not be a carball issue anymore...).

`ERROR: Could not find a version that satisfies the requirement boxcars-py==0.1.14 (from versions: 0.1.0, 0.1.1, 0.1.3, 0.1.4, 0.1.5, 0.1.6, 0.1.8, 0.1.9, 0.1.10, 0.1.12, 0.1.13)`

**Comment by beeftendon** [comment]
Was on python 3.6.15. Resolved the issue by switching to 3.8.12 and reinstalling all packages. Was able to install boxcars-py 0.1.14 this way.
